src/textual/reactive.py

```diff
@@ -203,7 +200,7 @@ def _reset_object(cls, obj: object) -> None:
         Args:
             obj: A reactive object.
         """
-        getattr(obj, "__watchers", {}).clear()
+        getattr(obj, "_watchers", {}).clear()
         getattr(obj, "__computes", []).clear()
```

Comment:
I feel we should either rename both `__watchers` and `__computes` to single underscore prefix or neither.

Reply:
Watchers was promoted because I needed to clear it from outside of the class. `__computes` didn't need that.

On reflection though, I can add a method to Reactive to do that, and I can keep the `__`